backend/src/services/comments/comment-utils.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const formatComments = comments =>
+  comments.map(comment => {
+    const { username, imageURL, ...rest } = comment;
+    rest.author = { username, imageURL };
```

Comment:
but how you will make a popup window (oh hover) with user information using only the image and username? maybe some additional information is needed?